src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -344,5 +349,20 @@ public string GetSdkFeatureBand()
         {
             return _sdkVersionBand.ToString();
         }
+
+        public static string? GetGlobalJsonPath(string? globalJsonStartDir)
+        {
+            string? directory = globalJsonStartDir;
```

Comment:
How does the SDK handle multiple global jsons in the directory? Do files closer to the end of the path you're executing in override the values that could've come from global jsons higher up the directories, or does it only look at the top level global json like we do here?

Reply:
Only one global.json ever applies, ie it starts in a folder and goes up the tree looking for global.json, and stops the search once it finds it.  It's not like NuGet.config where it will keep looking and merge the data from multiple files.